src/actions/index.js

```diff
@@ -25,6 +25,8 @@ import {
   STOP_SCHEDULER_FIXED_DELAY,
   STOP_SCHEDULER_FOR_RESUMING_NOTIFICATIONS,
   UPDATE_PAGING_DATA,
+  SET_DEFAULT_CONSOLE,
+  SET_VNC_MODE,
```

Comment:
Since we read those from config and we know that we'll probably override this by user settings in the future, I suggest to rename default_console and vnc_mode on all places in code to something like global-default-console and global-default-vnc-mode accordingly.